Features/Context/PlatformUI.php

```diff
@@ -408,33 +466,30 @@ protected function closeEditView()
         } catch (\Exception $e) {
         }
     }
+
     /**
-     * Attaches a file to a input field on the HTML.
+     * Returns an array with the properties contexts,
+     * if the properties use the Context Annotation.
      *
-     * @param   string $file        file name relative to mink definitions
-     * @param   string $selector    CSS file upload element selector
+     * @return array array of methods and their service dependencies
      */
-    protected function attachFile($fileName, $selector)
+    private function parseAnnotations($annotations)
     {
-        if ($this->getMinkParameter('files_path')) {
-            $fullPath = rtrim(
-                realpath(
-                    $this->getMinkParameter('files_path')
-                ),
-                DIRECTORY_SEPARATOR
-            ) . DIRECTORY_SEPARATOR . $fileName;
-
-            if (is_file($fullPath)) {
-                $fileInput = 'input[type="file"]' . $selector;
-                $field = $this->getSession()->getPage()->find('css', $fileInput);
-
-                if (null === $field) {
-                    throw new Exception("File input $selector is not found");
-                }
-                $field->attachFile($fullPath);
+        // parse array from (numeric key => 'annotation <value>') to (annotation => value)
+        $propertiesContexts = [];
+        foreach ($annotations as $annotation) {
+            if (!preg_match('/^(\w+)\s+\$(\w+)\s+([\w\.\\\\]+)/', $annotation, $matches)) {
+                continue;
+            }
+
+            array_shift($matches);
+            $tag = array_shift($matches);
+            if ($tag == 'Context') {
```

Comment:
The `Context` tag shouldn't really be necessary, 
An `@var` should suffice, as (see above) it should be checked if the class NS implements a `Context`